SpatialGDK/Documentation/content/whatsnew.md

```diff
@@ -1,25 +1,29 @@
 # What's new in alpha?
 
-If you’ve used the pre-alpha GDK you may be interested in these latest changes.
+If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.
 
 ## Iterative improvements
-- Upgraded the engine fork to support Unreal Engine 4.20.3.
-- Upgraded to the new SpatialOS load balancer, providing fast multi-worker iteration.
-- Removed the incremental build time overhead of using the GDK by implementing dynamic type bindings, avoiding the need to generate certain interop code. You can find more info [here]({{urlRoot}}/content/dynamic-typebindings)
-- Schema generation for all classes is now the default.
-- Clients no longer automatically connect to a deployment when not in PIE, unless you pass it a commandline argument. For example: 
-`"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor-Win64-Debug.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1`
-- Startup actors are now saved into the snapshot.
+- Upgraded the SpatialOS Unreal engine fork to support Unreal Engine 4.20.3.
+- Upgraded to the new SpatialOS load balancer, providing fast multi-[worker]({{urlRoot}}/content/glossary#workers) iteration.
+- Removed the incremental build-time overhead to using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This means you don't have to generate certain interop code. 
```

Comment:
```suggestion
- Removed the incremental build-time overhead of using the GDK by implementing [Dynamic Typebindings]({{urlRoot}}/content/glossary#dynamic-typebindings). This removes the additional code-generation and compile step.
```

Reply:
"The overhead to using" is better English. There is an overhead to doing something. I can change it if it makes everyone happier, though.

Reply:
I took me a few reads to understand what the sentence meant. But I grew up in Australia.